### DISCUSSION TITLE
How to show bar only on key-hold (as an overlay, i.e. above existing windows), then disappear on release (perhaps after a delay)?

### DISCUSSION THREAD

**Discussion by raj-magesh** [qtile/qtile#4189] [Q&A]
Not sure if this is possible, but it'd be a nice workflow IMO. If this isn't possible with the qtile bar, is there an alternative application that folks could suggest?

**Comment by justinesmithies** [comment]
Not exactly what you asked for but I toggle my bars using a key bind like below.

```
 # Toggle bars
    Key([mod], "b",
        lazy.hide_show_bar(position='all'),
        desc="Toggle bars"
        ),
```

**Reply by raj-magesh** [reply]
I do this too! Just wanted to see if I could get thie key-hold behavior to work.

**Comment by m-col** [comment]
There was some work done to support key-release bindings, which would enable this behaviour. The PR has gotten a bit stale though, so if there is renewed interest in this then contributions are welcome: https://github.com/qtile/qtile/pull/2785
